### ISSUE TITLE
Add Friends and Profile

### ISSUE THREAD

**Issue by ruymon** [ruymon/calypso#3]
-  [ ] Favorite users
- [ ] Show in separate list

**Comment by fonfs** [comment]
@ruymon essa é uma característica também meio que padrão das ferramentas concorrentes. Julgo importante ter já disponibilizado logo no lançamento.

Avaliar a complexidade.

**Comment by ruymon** [comment]
@ruymon Concordo plenamente. Tu pensa em alguma melhoria que podemos construir para sair na frente dos nossos  concorrentes?
